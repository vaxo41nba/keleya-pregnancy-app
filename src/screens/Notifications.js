import React from 'react';
import { Image, ImageBackground, StyleSheet, Text, View } from 'react-native';
import { t } from 'i18next';

import AppButton from '../components/AppButton';
import images from '../constants/images';
import colors from '../constants/colors';

export default function Notifications() {
  return (
    <ImageBackground
      source={images.notifications}
      style={{ flex: 1, alignItems: 'center' }}
    >
      <Image source={images.bell} style={styles.logo} />
      <Text style={styles.header}>
        {t('Get notifications to boost your motivation')}
      </Text>

      <View style={styles.footer}>
        <Text style={styles.loginText}>{t('Skip')}</Text>
        <AppButton text={t('Allow notifications')} />
      </View>
    </ImageBackground>
  );
}

const styles = StyleSheet.create({
  footer: {
    position: 'absolute',
    bottom: 25,
    left: 0,
    right: 0,
    alignItems: 'center',
  },
  header: {
    fontSize: 23,
    color: colors.GREYISH_BROWN,
    marginTop: 15,
    textAlign: 'center',
    marginHorizontal: '10%',
  },
  logo: { width: 40, height: 40, marginTop: 70, resizeMode: 'contain' },
  loginText: {
    color: colors.GREYISH_BROWN,
    fontSize: 22,
    marginVertical: 15,
  },
});
