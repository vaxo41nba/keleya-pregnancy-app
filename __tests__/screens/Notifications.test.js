import React from 'react';
import renderer from 'react-test-renderer';
import Notifications from '../../src/screens/Notifications';

test('renders correctly', () => {
  const tree = renderer.create(<Notifications />).toJSON();
  expect(tree).toMatchSnapshot();
});
