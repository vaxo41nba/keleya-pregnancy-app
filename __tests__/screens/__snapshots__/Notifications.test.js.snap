// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <Image
    source={1}
    style={
      [
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <Image
    source={1}
    style={
      {
        "height": 40,
        "marginTop": 70,
        "resizeMode": "contain",
        "width": 40,
      }
    }
  />
  <Text
    style={
      {
        "color": "#4a4a4a",
        "fontSize": 23,
        "marginHorizontal": "10%",
        "marginTop": 15,
        "textAlign": "center",
      }
    }
  />
  <View
    style={
      {
        "alignItems": "center",
        "bottom": 25,
        "left": 0,
        "position": "absolute",
        "right": 0,
      }
    }
  >
    <Text
      style={
        {
          "color": "#4a4a4a",
          "fontSize": 22,
          "marginVertical": 15,
        }
      }
    />
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#69c0ba",
          "borderRadius": 10,
          "height": 55,
          "justifyContent": "center",
          "opacity": 1,
          "paddingHorizontal": 10,
          "width": "80%",
        }
      }
    >
      <Text
        style={
          {
            "color": "#ffffff",
            "fontSize": 22,
            "textAlign": "center",
          }
        }
      />
    </View>
  </View>
</View>
`;
